draft-ietf-cdni-ci-triggers-rfc8007bis.xml

```diff
@@ -1218,568 +1044,686 @@ uCDN-A                         dCDN-B                         dCDN-C
+   Value: Non-empty array of JSON strings, where each string is a CDN PID as defined in <xref target="cdn-pid"/>.
    <vspace blankLines="1"/>
-   Mandatory: Yes.
+   Mandatory: No. The default is no CDN path.
    </t>
 
    <t hangText="Name: ctime">
    <vspace blankLines="1"/>
-   Description: The time at which the CI/T Command was received by the
-         dCDN. The time is determined by dCDN; there is no requirement
-         to synchronize clocks between interconnected CDNs.
+   Description: The time at which the trigger resource was received by the dCDN. 
+   The time is determined by dCDN; there is no requirement to synchronize clocks between interconnected CDNs.
    <vspace blankLines="1"/>
-   Value: Absolute Time, as defined in <xref target="absolute-time"/>.
+   Value: A JSON number, seconds since the UNIX epoch (00:00:00 UTC on 1 January 1970).
    <vspace blankLines="1"/>
    Mandatory: Yes.
    </t>
 
    <t hangText="Name: mtime">
    <vspace blankLines="1"/>
-   Description: The time at which the Trigger Status Resource was last
-         modified. The time is determined by dCDN; there is no
-         requirement to synchronize clocks between interconnected CDNs.
+   Description: The time at which the trigger resource was last modified. 
+   The time is determined by dCDN; there is no requirement to synchronize clocks between interconnected CDNs.
    <vspace blankLines="1"/>
-   Value: Absolute Time, as defined in <xref target="absolute-time"/>.
+   Value: A JSON number, seconds since the UNIX epoch (00:00:00 UTC on 1 January 1970).
    <vspace blankLines="1"/>
    Mandatory: Yes.
    </t>
 
    <t hangText="Name: etime">
    <vspace blankLines="1"/>
-   Description: The estimate of the time at which dCDN expects to
-         complete the activity. Time is determined by dCDN; there
-         is no requirement to synchronize clocks between interconnected
-         CDNs.
+   Description: The estimate of the time at which dCDN expects to complete the trigger processing. 
+   Time is determined by dCDN; there is no requirement to synchronize clocks between interconnected CDNs.
    <vspace blankLines="1"/>
-   Value: Absolute Time, as defined in <xref target="absolute-time"/>.
+   Value: A JSON number, seconds since the UNIX epoch (00:00:00 UTC on 1 January 1970).
    <vspace blankLines="1"/>
    Mandatory: No.
    </t>
 
-   <t hangText="Name: status">
+   <t hangText="Name: state">
    <vspace blankLines="1"/>
-   Description: Current status of the triggered activity.
+   Description: The current trigger state.
    <vspace blankLines="1"/>
-   Value: Trigger Status, as defined in <xref target="trigger-status"/>.
+   Value: Trigger state, as defined in <xref target="trigger-state"/>.
    <vspace blankLines="1"/>
-   Mandatory: Yes.
+   Mandatory: No.
    </t>
    
-   <t hangText="Name: status-reason">
+   <t hangText="Name: state-reason">
```

Comment:
This is true before your changes, but curiously, this field is never mentioned again except here, and does not appear in any of our examples.  Given that we have an "errors" section, do we feel like this is necessary, and if not, should we remove it?